packages/typespec-ts/src/utils/modelUtils.ts

```diff
@@ -298,17 +298,32 @@ function applyEncoding(
 
 function mergeFormatAndEncoding(
   format: string | undefined,
```

Comment:
it seems we duplicate the same function two different places. i prefer to keep it considering we have plan to split Modualr from RLC.